assets/scripts/dialogs/Analytics/SegmentAnalytics.js

```diff
@@ -1,81 +1,81 @@
-import Icon from '../../ui/Icon'
+import { FormattedMessage } from 'react-intl'
+import { getLocaleSegmentName } from '../../segments/view'
 
-const generateClass = (value) => {
-  if (value > 12000) {
-    return 'positive'
-  }
-
-  if (value > 5000) {
-    return 'warning'
-  }
+const BAR_HEIGHT = '70px'
+const BAR_MODIFIER = 0.85
+const BAR_COLORS = [
+  { main: '#1a914e', alt: '#c8edd1' },
+  { main: '#306c72', alt: '#dcf3f9' },
+  { main: '#b56d48', alt: '#f9d5c0' },
+  { main: '#536491', alt: '#d4dbe8' }
+]
 
-  return 'critical'
-}
-
-const PeopleGraph = ({ amount }) => {
-  const peeps = [...Array(Math.floor(amount / 1000))]
+// TODO replace placeholder 20000 max with the top value from the street?
+const CapacityBar = ({ amount, max, color }) => {
+  const widthPercent = `${(Number.parseInt(amount, 10) / max) * 100}%`
   return (
-    <span>
-      {peeps.map(() => { return (<Icon icon="person" />) })}
-    </span>
+    <div className="capacity-bar" style={{ height: BAR_HEIGHT, background: color, display: 'inline-block', width: widthPercent }} >&nbsp;</div>
   )
 }
 
-PeopleGraph.propTypes = { amount: PropTypes.number.isRequired }
+CapacityBar.propTypes = {
+  amount: PropTypes.number.isRequired,
+  max: PropTypes.number.isRequired,
+  color: PropTypes.string.isRequired
+}
 
-const SegmentAnalytics = ({ label, capacity, segment, locale }) => {
-  const [expanded, setCollapsed] = useState(true)
+const SegmentAnalytics = ({ type, capacity, segment, locale, index, chartMax }) => {
   const { average, potential } = capacity
-  const expandedEl = (
-    <React.Fragment>
-      <div>
-        {label}
-      </div>
-      <SegmentForPalette
-        onClick={() => setCollapsed(false)}
-        key={segment.id}
-        type={segment.type}
-        variantString={segment.variantString}
-      />
-      <div className="table-container">
-        <table>
-          <thead>
-            <td>Average</td>
-            <td>Potential</td>
-          </thead>
-          <tr>
-            <td className={generateClass(average)}>{FormatNumber(locale, average)}</td>
-            <td className={generateClass(potential)}>{FormatNumber(locale, potential)}</td>
-          </tr>
-        </table>
-        <div className="people-section">
-          <PeopleGraph amount={average} />
-          <PeopleGraph amount={potential - average} color={'#ccc'} />
-        </div>
-      </div>
-    </React.Fragment>
-  )
+  const label = getLocaleSegmentName(type, locale)
+  const color = BAR_COLORS[index % BAR_COLORS.length]
+  const capacityHeader = <div className="header">{label}</div>
+  const widthPercent = `${(Number.parseInt(potential, 10) / Number.parseInt(chartMax, 10)) * (100 * BAR_MODIFIER)}`
+  // leave 2% margin
+  const widthPercentInv = `${98 - Number.parseFloat(widthPercent)}`
+
+  const capacityUnits = (<div className="sub">
+    <FormattedMessage
+      id="analytics.streetSummary"
+      defaultMessage="{average} - {potential} people/hour"
```

Comment:
Replace the hyphen here with an "en dash" (used in typography to represent a range of numbers. See here for more info: https://practicaltypography.com/hyphens-and-dashes.html)